### PR TITLE
Captain radio key turns on all comms automatically

### PR DESCRIPTION
**Pull request by ynot01**
### Intent of your Pull Request

Captain radio key turns on all comms automatically

### Why is this change good for the game?

Reduces captain=security mindset very slightly

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 

Captain headset turns on all comms automatically
Reduces captain=security mindset very slightly

### What should players be aware of when it comes to the changes your PR is implementing?

Captain headset turns on all comms automatically

### What general grouping does this PR fall under? 

Command/captain/security

### Are there any aspects of the PR that you would like us not to mention on the Wiki?

No

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 

No

# Changelog

:cl:  
tweak: Captain encryption key turns on all comms automatically
/:cl:

### CODE REVIEW

**Comment by maxion12345** [comment]
I already turn on all channels as captain anyways :)

**Comment by alexkar598** [comment]
As if getting hold of the captain as a security officer to ask what the fuck they want to do with person X or Y isn't hard enough. Someone who wants to capcurity will turn on all channels anyways. Any captain that has atleast a single brain cell will turn on all channels anyways, security being just one of them. This just makes it harder to contact the captain as security without any reason for it.

The captain's main job is to help bridge communication holes between departments. Him not having all channels turned on by default is rather ridiculous anyways IMO. 

Another sec nerf/antag buff from ynot. How surprised I am.

**Comment by Tipy1802** [comment]
Literally why, captains already turn on all channels.
This is just a noob trap making it so first time captains can't tell sec who to arrest so they do it themself.
Won't lead to any reduction of capcuirty and maybe even unironically slightly promote it for the listed reason above

**Comment by JamieD1** [comment]
Probably better to turn on all channels instead to encourage other department interactivity instead.

**Review by JamieD1 — CHANGES_REQUESTED** [review]
Probably better to turn on all channels instead to encourage other department interactivity instead.

**Comment by ynot01** [comment]
> Probably better to turn on all channels

i consider this as a viable alternative for my goal but i fear it may overwhelm new captain players, though captains are supposed to be experienced anywayss

**Comment by Hopekz** [comment]
This PR pulled a complete 180 turn recently so it might be better to open a new one if you want feedback on your new changes.

**Comment by alexkar598** [comment]
hate to admit it, but I have to agree with hopek, changing what a pr does is wrong

**Comment by ynot01** [comment]
a change was requested, i made the change, im not opening a new pr

**Review by JamieD1 — APPROVED** [review]
